### PR TITLE
Add nuclear repulsion energy to docs

### PR DESCRIPTION
**Pull request by migatt**
This PR adds a small part of the nuclear method to the docs describing how to obtain the nuclear repulsion energy using cclib.

#1241

### CODE REVIEW

File: doc/sphinx/methods.rst [hunk 1]
```diff
@@ -450,6 +450,33 @@ After the calculate() method is called, the following attributes are available:
...
+    from cclib.method import Nuclear
+
+    parser = cclib.io.ccopen("test.log")
+    data = parser.parse()
+    nuclear = Nuclear(data)
+
+    nre = nuclear.repulsion_energy()
+    print(nre)
+
+The value can be also converted to Hartrees (to directly compare with Gaussian logfiles):
```
Comment (berquist) [inline doc/sphinx/methods.rst:473]: You've exposed a bug.  All energies that aren't excitation energies are supposed to be in eV (yeah, I know...), but your conversion snippet below does indeed produce the number in Hartrees.  That makes the result sort of doubly wrong.

I'd rather fix the units on the return value before updating these docs.  One way around that though is to fix the unit bug separately after merging this PR with the `convertor(...` snippet removed, and one can safely assume it has the same units as all other energies that are documented.

What do you think?

Comment (migatt) [reply doc/sphinx/methods.rst:473]: This sounds great.

**Review by shivupa — APPROVED** [review]
Unit fix can be in a following PR. Corresponding issue updated

**Review by berquist — APPROVED** [review]
Thanks for doing all of this!

**Pull request merged**